src/lib/crypto/rsa.cpp

```diff
@@ -369,6 +369,10 @@ rsa_generate(rng_t *rng, pgp_rsa_key_t *key, size_t numbits)
         goto end;
     }
 
+    if (botan_privkey_create(
+          &rsa_key, "RSA", std::to_string(numbits).c_str(), rng_handle(rng))) {
+        goto end;
+    }
     if (botan_privkey_create_rsa(&rsa_key, rng_handle(rng), numbits) != 0) {
```

Comment:
Should not this be removed since replaced with previous `botan_privkey_create()` call?

Reply:
Oops, good catch!